src/main/java/core/basesyntax/modul/DrawingSystem.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.modul;
+
+public interface DrawingSystem {
+    void toDraw(double area);
+}
```

Comment:
```suggestion
public interface DrawingSystem {
    void draw();
}
```

we could get area directly from class